android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -407,218 +413,360 @@ public void testClassesExist() throws Exception {
+                        null));
+
+        Assert.assertTrue(
+                methodExists(
+                        "org/chromium/chrome/browser/ntp/NewTabPageLayout",
+                        "isScrollableMvtEnabled",
+                        MethodModifier.REGULAR,
+                        true,
+                        boolean.class));
+
         Assert.assertTrue(
                 methodExists(
                         "org/chromium/chrome/browser/search_engines/settings/SearchEngineAdapter",
                         "getSearchEngineSourceType",
+                        MethodModifier.STATIC,
                         false,
                         null));
         Assert.assertTrue(
                 methodExists(
                         "org/chromium/chrome/browser/search_engines/settings/SearchEngineAdapter",
                         "sortAndFilterUnnecessaryTemplateUrl",
+                        MethodModifier.STATIC,
+                        false,
+                        null));
+        Assert.assertTrue(
+                methodExists(
+                        "org/chromium/base/CommandLineInitUtil",
+                        "initCommandLine",
+                        MethodModifier.STATIC,
                         false,
                         null));
-        Assert.assertTrue(methodExists(
-                "org/chromium/base/CommandLineInitUtil", "initCommandLine", false, null));
-        Assert.assertTrue(methodExists(
-                "org/chromium/chrome/browser/ui/appmenu/AppMenu", "getPopupPosition", false, null));
-        Assert.assertTrue(methodExists("org/chromium/chrome/browser/ui/appmenu/AppMenu",
-                "runMenuItemEnterAnimations", false, null));
-        Assert.assertTrue(methodExists(
-                "org/chromium/chrome/browser/ui/default_browser_promo/DefaultBrowserPromoUtils",
-                "prepareLaunchPromoIfNeeded", false, null));
+
+        Assert.assertTrue(
+                methodExists(
+                        "org/chromium/chrome/browser/ui/appmenu/AppMenu",
+                        "getPopupPosition",
+                        MethodModifier.STATIC,
+                        false,
+                        null));
+
+        Assert.assertTrue(
+                methodExists(
+                        "org/chromium/chrome/browser/ui/appmenu/AppMenu",
+                        "runMenuItemEnterAnimations",
+                        MethodModifier.REGULAR,
+                        false,
+                        null));
+        Assert.assertTrue(
+                methodExists(
+                        "org/chromium/chrome/browser/ui/"
+                                + "default_browser_promo/DefaultBrowserPromoUtils",
+                        "prepareLaunchPromoIfNeeded",
+                        MethodModifier.STATIC,
+                        false,
+                        null));
+
+        // Why do we have this below? we can guarantee whatever we need at Brave code
```

Comment:
@samartnik , I forgot why do we need the `methodExists` tests for `Brave*****` classes. This looks suspicious here and below in several places. Can you please remind me why we did so?

Reply:
yes, this seems to be overkill, the main point of these tests is to detect changes in the upstream

Reply:
thanks for the confirmation @samartnik , I am updating the PR.